blog-post.html

```diff
@@ -1,12 +1,76 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>Blog Post</title>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-  </head>
-  <body>
-    <section id="header"></section>
-    <section id="main"></section>
-  </body>
-</html>
+
+<head>
+  <title>Blog Post</title>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+</head>
+<style>
+  h1 {
+    text-align: center;
+    font: size 60px;
+    font-family: "Notable", sans-serif;
+  }
+
+  .home-btn {
+    background-color: purple;
+    color: white;
+    cursor: pointer;
+  }
+
+  .home-btn:active {
+    color: blue
+  }
+
+  .home-btn:hover {
+    background-color: rgb(63, 61, 61);
+  }
+</style>
+
+<body>
+  <section id="header">
+    <h1>REACT!</h1>
+  </section>
+  <form action="index.html">
+    <button class="home-btn">HOME</button>
+  </form>

```

Comment:
This is an interesting way to do this but using an `<a>` tag here might be more appropriate since we aren't submitting anything, we're just trying to change the URL